tensorflow/contrib/framework/python/framework/graph_util_test.py

```diff
@@ -57,5 +60,17 @@ def testGraphUtil(self):
     self.assertEqual(fused_graph_def.node[3].name, 'E')
 
 
+class GetPlaceholdersTest(test.TestCase):
+
+  def test_get_placeholders(self):
+    with ops.Graph().as_default() as g:
+      ids = []
+      for i in range(5):
+        x = array_ops.placeholder(dtypes.float32)
+        ids.append(x._id)
+      result = graph_util.get_placeholders(g)
+      result_ids = [ i._id for i in result ]
+      self.assertEqual(sorted(result_ids), sorted(ids))
+
```

Comment:
Perhaps [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html) could be useful to check the code style.